README.md

```diff
@@ -1,5 +1,8 @@
 # BlueSky : Web dev in Eclipse IDE
 
+[![Eclipse License](http://img.shields.io/badge/license-Eclipse-brightgreen.svg)](https://github.com/mickaelistria/eclipse-bluesky/blob/master/LICENSE)
```

Comment:
For licenses, we need to be precise: there is no such thing as `Eclipse License`. It's currently `EPL v1` (or `Eclipse Public License v1` if we want longer names)